nistats/design_matrix.py

```diff
@@ -274,7 +281,7 @@ def _full_rank(X, cmax=1e15):
 def make_design_matrix(
     frame_times, paradigm=None, hrf_model='glover',
     drift_model='cosine', period_cut=128, drift_order=1, fir_delays=[0],
-    add_regs=None, add_reg_names=None, min_onset=-24):
+    add_regs=None, add_reg_names=None, min_onset=-24, oversampling=None):
```

Comment:
In some functions oversampling is by default 16. Is there any reason to keep default to None and then convert to 16.